litex/soc/cores/cpu/coreblocks/core.py

```diff
@@ -0,0 +1,148 @@
+#
+# This file is part of LiteX.
+#
+# Copyright (c) 2023-2025 Piotr Wegrzyn (Coreforge Foundation) <piotro@piotro.eu>
+# SPDX-License-Identifier: BSD-2-Clause
+
+import os
+import subprocess
+
+from migen import *
+
+from litex import get_data_mod
+from litex.soc.cores.cpu import CPU, CPU_GCC_TRIPLE_RISCV32
+from litex.soc.interconnect import wishbone
+
+# Variants -----------------------------------------------------------------------------------------
+
+CPU_VARIANTS = {
+    "minimal":  "tiny",
+    "standard": "basic",
+    "full":     "full",
+}
+
+# GCC Flags ----------------------------------------------------------------------------------------
+
+GCC_FLAGS = {
+    "minimal":          "-march=rv32e2p1                                      -mabi=ilp32 ",
```

Comment:
`rv32i2p1` ?

Reply:
`rv32e2p1` is correct, this is Embedded extension that substitutes base `i`. I missed that abi should be `-mabi=ilp32e` instead.

Reply:
Indeed